project/Build.scala

```diff
@@ -63,7 +63,7 @@ object MyScalaJSPlugin extends AutoPlugin {
 }
 
 object Build {
-  val referenceVersion = "3.0.0-RC2-bin-20210318-e60ef35-NIGHTLY"
+  val referenceVersion = "3.0.0-RC2-bin-20210325-ab2664f-NIGHTLY"
```

Comment:
We can also enable `-Yno-experimental` for the compiler (search for `TODO` in the file).

Reply:
Done.

Reply:
I reverted it because I got some `Flag -Yno-experimental set repeatedly` which seem to indicate that we might not need those or need to remove some others. Do you know which ones could be conflicting? All other use cases seem to be set in the tests.

Reply:
Seems like the one in `commonDottyCompilerSettings` can be enabled. The other one causes the `Flag -Yno-experimental set repeatedly` even if it is not set in `commonDottyCompilerSettings`.

Reply:
That's a bit weird, we can investigate later.